### PR TITLE
Update feature page template

### PR DESCRIPTION
**Pull request by yanivlavi**
Updated feature page template to make it easier to document.

Changes proposed in this pull request:
- Changed description for summary, detailed description, installation/upgrade sections.
- Added prerequisites and limitations sections.

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): (please @dyaniv yourself to sign)

This pull request needs review by: @mykaul @JohnMarksRH

### CODE REVIEW

File: source/feature/feature-template.html.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ The actual name of your feature page should look something like: "Your feature n
 
 ### Summary
 
-A sentence or two summarizing what this feature is and what it will do. This information is used for the overall feature summary page for each release.
+Summarize what this feature is and what it will do. Explain to the reader why this feature is necessary, and what problem it is trying to solve. This information is used for the overall feature summary page for each release.
```
Comment (mykaul) [inline source/feature/feature-template.html.md]: why this feature is necessary ->  what is the benefit to the end user

File: source/feature/feature-template.html.md [hunk 2]
```diff
@@ -34,7 +34,13 @@ Include you email address that you can be reached should people want to contact
 
 ### Detailed Description
 
-Expand on the summary, if appropriate. A couple sentences suffices to explain the goal, but the more details you can provide the better.
+Expand on the summary, if appropriate. Explain the goal; the more details you can provide the better. 
+
+### Prerequisites
+List any hardware or software prerequisites. Or any steps that need to be performed before the feature can be implemented.
```
Comment (mykaul) [inline source/feature/feature-template.html.md]: ". Or" -> "or"

File: source/feature/feature-template.html.md (hunk 1 above)
Comment (dyaniv) [inline source/feature/feature-template.html.md]: Done

File: source/feature/feature-template.html.md (hunk 2 above)
Comment (dyaniv) [inline source/feature/feature-template.html.md]: Done

**Comment by doron-fediuck** [comment]
Please do not merge before asking me.
Thanks!

File: source/feature/feature-template.html.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ Include you email address that you can be reached should people want to contact
 Expand on the summary, if appropriate. Explain the goal; the more details you can provide the better. 
 
 ### Prerequisites
-List any hardware or software prerequisites. Or any steps that need to be performed before the feature can be implemented.
+List any hardware or software prerequisites or any steps that need to be performed before the feature can be implemented.
```
Comment (Adam-Weiss) [inline source/feature/feature-template.html.md]: "**and** any steps is more curate as both  hardware \ software prerequisites and future steps must be listed.

File: source/feature/feature-template.html.md [hunk 4]
```diff
@@ -37,7 +37,7 @@ Include you email address that you can be reached should people want to contact
 Expand on the summary, if appropriate. Explain the goal; the more details you can provide the better.
```
Comment (Adam-Weiss) [inline source/feature/feature-template.html.md]: more details you can provide the better -> be as detailed as possible. (The instruction is clearer and more concise,)

File: source/feature/feature-template.html.md [hunk 5]
```diff
@@ -34,7 +34,13 @@ Include you email address that you can be reached should people want to contact
 
 ### Detailed Description
 
-Expand on the summary, if appropriate. A couple sentences suffices to explain the goal, but the more details you can provide the better.
```
Comment (Adam-Weiss) [inline source/feature/feature-template.html.md]: goal -> purpose

File: source/feature/feature-template.html.md [hunk 6]
```diff
@@ -34,7 +34,13 @@ Include you email address that you can be reached should people want to contact
...
+Expand on the summary, if appropriate. Explain the goal; the more details you can provide the better.
```
Comment (JohnMarksRH) [inline source/feature/feature-template.html.md]: Suggest to change to:
"Expand on the summary. Make sure the goal is clearly explained. The more detail you can provide, the better."

File: source/feature/feature-template.html.md [hunk 7]
```diff
@@ -55,7 +61,7 @@ Describe user experience related issues. For example: We need a wizard for ....,
 
 #### Installation/Upgrade
 
-Describe how the feature will effect new installation or existing one.
+Describe how this feature will affect new installations or existing ones.
```
Comment (JohnMarksRH) [inline source/feature/feature-template.html.md]: Suggest to change to:
"Describe how this feature will affect new or existing installations."

Comment (Adam-Weiss) [inline source/feature/feature-template.html.md]: Right!

**Comment by JohnMarksRH** [comment]
This document still has some outstanding issues to be addressed:

Your Feature's name: Can we say something more specific than ....look something like "your feature name"?

Subheadings: There are both H3 and H4 headings, but the hierarchical relationship between them is not clear.

Release notes - I think it requires clarification

Open issues: I think it requires clarification

**Comment by bgraveno** [comment]
We should probably mention that the Feature Name should be meaningful, and as much as possible, describe what the feature is about.

**Comment by JohnMarksRH** [comment]
@dyaniv @bgraveno The current feature page is based on PR#979. Regarding the feature name, I tweaked the text and instead of using "meaningful" I tried to be more specific: "short, clear, and not overly technical." I will add "descriptive" . See PR#979

**Comment by doron-fediuck** [comment]
@yanivlavi @JohnMarksRH what about this PR? It's waiting for too long and I suggest to fix and merge it or  abandon.

**Comment by JohnMarksRH** [comment]
@doron-fediuck @yanivlavi Looks like we implemented the changes in a different PR. We can close this one.

**Pull request closed without merging**